src/UI/Series/series.less

```diff
@@ -469,3 +469,14 @@
     list-style-type : none;
   }
 }
+
+//Series footer
+.break-sm {
+  font-style: normal;
+}
+
+@media only screen and (max-width : 600px) {
```

Comment:
`max-width:`
Use `@screen-sm-max` or `@screen-xs-max` (for closer to 600px) instead of a custom width.